### PR TITLE
WIP: Update to TestNG 7.1.0

### PR DESCRIPTION
**Pull request by wendigo**
Just a test for now :)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1199,6 +1199,22 @@
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
```
Comment (wendigo) [inline pom.xml:1202]: It should be updated on airbase

**Comment by findepi** [comment]
What does it bring us?

**Comment by kokosing** [comment]
> What does it bring us?

IIRC it allows to define tests in `interface`.

**Comment by findepi** [comment]
> IIRC it allows to define tests in `interface`.

very cool. does it work with BeforeClass, BeforeTest, annotations respecting especially `After..(alwaysRun=true)`?

**Comment by wendigo** [comment]
@findepi we have to wait for 7.1.1 which has guice bumped to version we are using. So this PR will be stale for a while.

**Pull request closed without merging**